spec/hashTable.js

```diff
@@ -0,0 +1,171 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import HashTable from '../src/hashTable'
+
+chai.use( chaiChange )
+
+describe( 'HashTable', () => {
```

Comment:
the mocha docks strongly recommend you use `describe('HashTable', function(){ …`